applications/API/errors.py

```
@@ -0,0 +1,472 @@
+    status.HTTP_404_NOT_FOUND: ERROR_TEMPLATES[status.HTTP_404_NOT_FOUND],
+    status.HTTP_500_INTERNAL_SERVER_ERROR: ERROR_TEMPLATES[status.HTTP_500_INTERNAL_SERVER_ERROR],
+}
+
+
+def yo_momma_exception_handler(exc: Exception, context: Dict[str, Any]) -> Response:
+    """
+    Reusable exception handler for The Dozens API.
+
+    - Delegates to DRF's default exception handler first.
+    - Normalizes the response body shape into {detail, code, status_code, ...}.
+    - Injects Yo Momma–style messages for selected HTTP status codes.
```

Ruff (0.14.5)

Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)